systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaNodePoolST.java

```diff
@@ -212,7 +212,7 @@ void setup(ExtensionContext extensionContext) {
 
         clusterOperator = new SetupClusterOperator.SetupClusterOperatorBuilder()
             .withExtensionContext(extensionContext)
-            .withNamespace(INFRA_NAMESPACE)
+            .withNamespace(CO_NAMESPACE)
```

Comment:
Everywhere we use `Constants.CO_NAMESPACE`, why not here?

Reply:
True. I will update it :) Thanks